go.mod

```diff
@@ -127,31 +127,31 @@ replace (
 	google.golang.org/grpc => google.golang.org/grpc v1.27.1
 
 	// Kubernetes
-	k8s.io/api => k8s.io/api v0.21.3
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.21.3
-	k8s.io/apimachinery => k8s.io/apimachinery v0.21.3
-	k8s.io/apiserver => k8s.io/apiserver v0.21.3
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.21.3
-	k8s.io/client-go => k8s.io/client-go v0.21.3
-	k8s.io/cloud-provider => k8s.io/cloud-provider v0.21.3
-	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.21.3
-	k8s.io/code-generator => k8s.io/code-generator v0.21.3
-	k8s.io/component-base => k8s.io/component-base v0.21.3
-	k8s.io/component-helpers => k8s.io/component-helpers v0.21.3
-	k8s.io/controller-manager => k8s.io/controller-manager v0.21.3
-	k8s.io/cri-api => k8s.io/cri-api v0.21.3
-	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.21.3
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.21.3
-	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.21.3
-	k8s.io/kube-proxy => k8s.io/kube-proxy v0.21.3
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.21.3
-	k8s.io/kubectl => k8s.io/kubectl v0.21.3
-	k8s.io/kubelet => k8s.io/kubelet v0.21.3
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.21.3
-	k8s.io/metrics => k8s.io/metrics v0.21.3
-	k8s.io/mount-utils => k8s.io/mount-utils v0.21.3
-	k8s.io/node-api => k8s.io/node-api v0.21.3
-	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.21.3
-	k8s.io/sample-cli-plugin => k8s.io/sample-cli-plugin v0.21.3
-	k8s.io/sample-controller => k8s.io/sample-controller v0.21.3
+	k8s.io/api => k8s.io/api v0.21.9
```

Comment:
Question: why are we replacing to the same version as defined in the requirement?
Is there any reason not to drop the replacement where the required version is already the same?

Reply:
Well, as I could see in previous versions this was always the case so I looked at it as a collection of all k8s libs to ensure they are all on the same version, without checking what they are replacing (easier to maintain). But that is just something I assumed, do you think it would be cleaner to drop the ones that are already on the same version?

Reply:
I think we are double booking now. Once at the require and secondly at the replace. I concur it sometimes more clear to see the replacements be on the same version than the implicit requirements so we can leave it in, thanks.